ITK_dev_shared_components/SAP/gridview_util.py

```diff
@@ -97,6 +97,36 @@ def find_row_index_by_value(grid_view, column:str, value:str) -> int:
             return row
     
     return -1
+
+def find_all_row_indecies_by_value(grid_view, column:str, value:str) -> list[int]:
+    """Find all indecies of the rows where the given column's value
+    match the given value. Returns an empty list if no row is found.
+
+    Args:
+        grid_view (_type_): A SAP GuiGridView object.
+        column (str): The name of the column whose value to check.
```

Comment:
I would rephrase the row from " The name of the column whose value to check."
to 
"[...]whose rows to check."